deploy/k8s-onprem/values.yaml

```diff
@@ -27,6 +27,7 @@
 tags:
   autoscaling: true
   loadBalancing: true
+  openshift: false
```

Comment:
Can we get some descriptive comment explaining what all of these values actually do and what the expected values for them are?

We keep publishing unusable interfaces and this is a perfect example. I am sure it makes total sense to the authors of this file, but anyone coming in cold is mostly left clueless and guessing at best. That's a terrible user experience.